internal/backend/discussion.go

```diff
@@ -27,14 +30,42 @@ func (d *delphisBackend) CreateNewDiscussion(ctx context.Context, creatingUser *
 	}
 
 	discussionID := util.UUIDv4()
+	now := time.Now()
+	titleHistory := []model.HistoricalString{
+		{
+			Value:     title,
+			CreatedAt: now,
+		},
+	}
+	descriptionHistory := []model.HistoricalString{
+		{
+			Value:     description,
+			CreatedAt: now,
+		},
+	}
+	titleHistoryBytes, err := json.Marshal(titleHistory)
+	if err != nil {
+		// Not sure what to do here?
```

Comment:
we should return an error. it shouldn't hit this as we are marshaling a struct but it would be safer to return if we hit an error